app/src/renderer/components/wallet/SendModal.vue

```diff
@@ -0,0 +1,297 @@
+        v-model.number="$v.fields.amount.$model"
+        type="number"
+        placeholder="Amount"
+      />
+      <tm-form-msg
+        v-if="
+          $v.fields.amount.$dirty &&
+            $v.fields.amount.$invalid &&
+            !$v.fields.amount.required
+        "
+        name="Amount"
+        type="required"
+      />
+      <tm-form-msg
+        v-else-if="!$v.fields.amount.between"
+        :max="$v.fields.amount.$params.between.max"
+        :min="$v.fields.amount.$params.between.min"
+        name="Amount"
+        type="between"
+      />
+      <tm-form-msg
+        v-else="!$v.fields.amount.integer"
+        name="Amount"
+        type="integer"
+      />
+    </tm-form-group>
+
+    <tm-form-group
+      :error="
+        fields.password.length > 0 &&
+          ($v.fields.password.$dirty && $v.fields.password.$invalid)
+      "
+      field-id="password"
+      field-label="Password"
+    >
+      <tm-field
+        id="password"
+        v-model="$v.fields.password.$model"
+        type="password"
+        placeholder="Password"
+      />
+      <tm-form-msg
+        v-if="
+          $v.fields.password.$dirty &&
+            fields.password.length &&
+            $v.fields.password.$invalid &&
+            !$v.fields.password.required
+        "
+        name="Password"
+        type="required"
+      />
+    </tm-form-group>
+
+    <div slot="action-modal-footer">
+      <tm-btn
+        v-if="sending"
+        value="Sending..."
+        disabled="disabled"
+        color="primary"
+      />
+      <tm-btn
+        v-else-if="!connected"
+        value="Connecting..."
+        disabled="disabled"
+        color="primary"
+      />
+      <tm-btn
+        v-else
+        id="send-btn"
+        value="Send Tokens"
+        color="primary"
+        :disabled="$v.fields.$invalid"
+        @click.native="validateForm"
+      />
+    </div>
+  </action-modal>
+</template>
+
+<script>
+import b32 from "scripts/b32"
+import { required, between, integer } from "vuelidate/lib/validators"
+import { mapActions, mapGetters } from "vuex"
+import TmBtn from "common/TmBtn"
+import TmFieldGroup from "common/TmFieldGroup"
+import TmFormGroup from "common/TmFormGroup"
+import TmFormStruct from "common/TmFormStruct"
+import TmPage from "common/TmPage"
+import TmPart from "common/TmPart"
+import TmField from "common/TmField"
+import TmFormMsg from "common/TmFormMsg"
+import TmBalance from "common/TmBalance"
+import FieldAddon from "common/TmFieldAddon"
+import ToolBar from "common/ToolBar"
+import ActionModal from "common/ActionModal"
+
+export default {
+  name: `send-modal`,
+  components: {
+    TmBalance,
+    TmBtn,
+    TmField,
+    FieldAddon,
+    TmFieldGroup,
+    TmFormGroup,
+    TmFormMsg,
+    TmFormStruct,
+    TmPage,
+    TmPart,
+    ToolBar,
+    ActionModal
+  },
+  props: {
+    denom: {
+      type: String,
+      default: ``
+    }
+  },
+  data: () => ({
+    bech32error: null,
+    fields: {
```

Comment:
I'd remove this object, it's unnecessary imo ...

Reply:
yeah probably we should do that but it's a small refactor - might need to wait for another time.